TechTalk.JiraRestClient/packages.config

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is it necessary to update RestSharp reference?
If it is, then update the .nupkg file as well.